### ISSUE TITLE
capturing DDL operations ?

### ISSUE THREAD

**Issue by AvinashUrs** [erdemcer/kafka-connect-oracle#33]
Hi,
May i know how difficult or how much effort does it takes to implement DDL operations as well?

**Comment by erdemcer** [comment]
Hi,
I think it is not so difficult to implement it . After PR merge which is waiting , it can be done.
Thanks

**Comment by AvinashUrs** [comment]
> Hi,
> I think it is not so difficult to implement it . After PR merge which is waiting , it can be done.
> Thanks

Thanks for your response. Also, when can we expect the PR merge to be done?

**Comment by erdemcer** [comment]
I have a target to finish it in 2 weeks

**Comment by AvinashUrs** [comment]
Than you.

**Comment by louisliu318** [comment]
Looking forward to the new feature!

**Comment by yongorg** [comment]
I'm really looking forward to this feature being implemented quickly

**Comment by yongorg** [comment]
Come on !!! Come on!!!

**Comment by erdemcer** [comment]
Hi,
I have merged PR and began to DDL implementation. I will update asap.
Thanks

**Comment by yongorg** [comment]
Hi,How long will it take.I'm looking forward to it

**Comment by erdemcer** [comment]
Yes , i know but  because of some unexpected issues took my time to complete DDL support.Also  I would like to add this property to my connector . 
What is your db version ?

**Comment by yongorg** [comment]
thanks, is 11g. Is it related to db version?

**Comment by erdemcer** [comment]
Hi,
While using connector , do you collect all DMLs in 1 topic or multiple topic for each table ?

**Comment by yongorg** [comment]
hi，
I am collecting all the tables into one topic.  Is that a problem?  thanks！

**Comment by erdemcer** [comment]
Hi,
Some DDL statements does not include segment_name  . Therefore if you configure connector for multiple topics for each table , these DDLs can not be captured for related table topic. This can be problem.  But 1 topic configuration , this is not problem . All DDLs goes into 1 topic with other DMLs. 
Firstly i will update connector  for DDL support for 1 topic configuration which is your configuration.

**Comment by yongorg** [comment]
thanks，Can I capture DML based on the timestamp range?For example, I want to capture DML by marking the time of the start and end points.Can this connector be implemented?In the future?

**Comment by yongorg** [comment]
Thank you very much for your help!

**Comment by erdemcer** [comment]
Hi,
I have updated code with DDL capture support ? Could you please test it ? I have written some explanation on README .
Capturing DML statements for time range can be done . In the future it can be taken into to do list.
Thanks

**Comment by yongorg** [comment]
Thanks!
